### ISSUE TITLE
floating point exception (core dumped)

### ISSUE THREAD

**Issue by pitje** [Komet/MediaElch#593]
when scraping for info for (a) TV show(s), MediaElch will give result after searching for a show, but will crash after selecting the show.
This has happened before (a year or so ago), but, after about a week (or two), 'suddenly' ceased to happen, without updating MediaElch. Everything was fine until about a month ago, when MediaElch crashed while updating a show. It continues to happen to this day, forcing me to not update my kodi :(

I'm running MediaElch on Linux (archlabs linux, fully up-to-date)
If I start MediaElch (why the CamelCase in the executable?) from cli, this is what I see:

<snip>
                                                                       Entered, with "id255316"
                                                                       Entered
                                                                       Entered, searchStr= "id255316"
                                                                       WARNING: Network Error "Operation canceled"
                                                                       Entered, size of results= 0
                                                                       Entered, with "Elementary"
                                                                       Entered
                                                                       Entered, searchStr= "Elementary"
                                                                       Entered, size of results= 4
                                                                       Entered
                                                                       m_scraperId= "255316"
                                                                       Entered
                                                                       Entered, m_scraperId "255316"
                                                                       Entered, id= "255316" scraperInterface= "The TV DB"
                                                                       Entered, id= "255316" episode= "S06E19 mkv"
[1]    4338 floating point exception (core dumped)  MediaElch

**Comment by bugwelle** [comment]
> I've tried the AppImage version, but that crashes right away

That shouldn't happen... I'll install archlabs linux as well and will test it. :)

**Comment by bugwelle** [comment]
Yeah, the AppImage fails. I've opened: https://github.com/Komet/MediaElch/issues/596
